test/integration/client/exec_test.go

```diff
@@ -125,7 +125,7 @@ func execPluginClientTests(t *testing.T, unauthorizedCert, unauthorizedKey []byt
 						Name: outputEnvVar,
 						Value: `{
 										"kind": "ExecCredential",
-										"apiVersion": "client.authentication.k8s.io/v1beta1",
+										"apiVersion": "client.authentication.k8s.io/v1",
```

Comment:
We probably want at least some beta tests in here?

Reply:
hoping https://github.com/kubernetes/kubernetes/pull/102890/commits/34ffddfc7a905e5aacf8e127e9d78ab458646f45 adds enough beta coverage